rubicon_ml/viz/metric_lists_comparison.py

```diff
@@ -78,7 +78,7 @@ def load_experiment_data(self):
 
         for experiment in self.experiments:
             for metric in experiment.metrics():
-                if isinstance(metric.value, list):
+                if isinstance(metric.value, float):
```

Comment:
we shouldn't be changing anything in this file - this needs to stay a list. you'll need to log list-type metrics to test if that's why this was changed